tensorflow/compiler/mlir/xla/tests/lhlo-legalize-to-linalg.mlir

```diff
@@ -649,3 +649,28 @@ func @reverse(%arg0: memref<2x3xf32>, %arg1: memref<2x3xf32>) {
   return
 }
 // CHECK: linalg.generic {{{.*}}indexing_maps = [#[[OPERAND_MAP]], #[[RESULT_MAP]]]
+
+
+// -----
+
+func @conv(%input: memref<3x5x5x3xf32>, %filter: memref<2x2x3x4xf32>, %output: memref<3x5x5x4xf32>) {
+  %c0 = constant 0 : index
+  %0 = alloc() : memref<3x5x5x4xf32>
+  // CHECK: linalg.conv(%{{.+}}, %{{.+}}, %{{.+}})
+  // CHECK-SAME: dilations = [1, 2]
+  // CHECK-SAME: padding = dense<[
+  // CHECK-SAME:                  [0, 1], [0, 1]]> : tensor<2x2xi64>
```

Comment:
I'm not sure why the two lines above aren't fused?

Reply:
It causes FileCheck to treat the thing in [[ ]] as ...

Reply:
It can be escaped: `{{\[\[}}`

(the two lines here isn't semantically equivalent as there could be other things in between the two checks)

Reply:
Yes, but I thought it wasn't as readable with that - but I do note the other point you make. Changed now to escape.